src/utils.js

```diff
@@ -901,14 +901,16 @@ export function toPathSchema(schema, name = "", definitions, formData = {}) {
     });
   } else if (schema.hasOwnProperty("properties")) {
     for (const property in schema.properties) {
-      pathSchema[property] = toPathSchema(
-        schema.properties[property],
-        `${name}.${property}`,
-        definitions,
-        // It's possible that formData is not an object -- this can happen if an
-        // array item has just been added, but not populated with data yet
-        (formData || {})[property]
-      );
+      if (schema.properties.hasOwnProperty(property)) {
```

Comment:
Is there an example of a schema in which the original code would fail? Could you add a test with it?

Reply:
https://stackoverflow.com/questions/12735778/for-in-and-hasownproperty